lib/rules/kind-name.js

```diff
@@ -53,13 +53,7 @@ module.exports = {
         case 'ClassProperty':
         case 'PropertyDefinition':
           var tokens = sourceCode.getFirstTokens(node, 2);
-          if (
-            tokens[0].value === 'name' ||
-            (tokens[1] && tokens[1].value === 'name')
-          ) {
-            return true;
-          }
-          return false;
+          return !!(tokens[0].value === 'name' || (tokens[1] && tokens[1].value === 'name'));
```

Comment:
How about keeping the original code? The logic is the same but for better readablility.